packages/theme-patternfly-org/helpers/sendEvent.js

```diff
@@ -0,0 +1,8 @@
+export const sendEvent = (name, eventCategory, eventLabel, value) => {
```

Comment:
Just to play on the title of this PR, I wonder if this function makes more sense to be named `trackEvent`.

Also, a docstring would be useful here to describe each of the parameters.

Reply:
Also, is there any reason to not make this a typescript file so we can type each of the params as well?

Reply:
@jeffpuzzo thanks for the feedback!  I went ahead and renamed the function per your suggestion.

When converting to TS it broke the build, it looks like we don't actually have TS running in the Org repo right now.  That's outside of the scope of this issue, and of my expertise, so I've reverted it back to JS for now and updated the file to better document the function & params.

Reply:
Good suggestion - created https://github.com/patternfly/patternfly-org/issues/2846 to track this 👍